SPECS/postgresql/postgresql.spec

```diff
@@ -8,7 +8,7 @@ Distribution:   Azure Linux
 Group:          Applications/Databases
 URL:            https://www.postgresql.org
 Source0:        https://ftp.postgresql.org/pub/source/v%{version}/%{name}-%{version}.tar.bz2
-
+Source1:        %{name}.service
```

Comment:
We have missed to bump the release number.

Reply:
Done